static/_redirects

```diff
@@ -1,8 +1,9 @@
+https://docs.metricly.com/* https://docs.virtana.com//:splat 301!
```

Comment:
This line I think will break all the lines below once you merge it in.

Reply:
removed the line